scripts/main/lychee.js

```diff
@@ -509,7 +509,7 @@ lychee.localizeStaticGuiElements = function () {
 					: lychee.site_copyright_begin + "–" + lychee.site_copyright_end
 		  )
 		: "";
-	footer.querySelector("p.personal_text").textContent = lychee.footer_additional_text;
+	footer.querySelector("p.personal_text").innerHTML = lychee.footer_additional_text;
```

Comment:
that part is no go as it potentially allows XSS.

Reply:
done, I would appreciate some advice on this:

https://github.com/LycheeOrg/Lychee-front/pull/375